clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriter.java

```diff
@@ -337,21 +343,6 @@ private boolean verifyChecksum(String sstFilePath, long expectedRecordNumInSSTFi
     }
   }
 
-  public boolean validateBatchIngestion() {
```

Comment:
@majisourav need your inputs on removing this. This was added as a part of this [commit](https://github.com/linkedin/venice/commit/ad6cf8c9b15e0d17a3a5d38e9a821ac8f842b444).

Reply:
can we validate just before sst files are removed ? It was added after a GCN where we did not ingest a bunch of sst files due to some bug.

Reply:
I will sync up with you on this. I am still trying to figure out is there a way to figure out validate whether the src sst files and dst sst files are same meaning the ingestion is successful. But as per the documentations, 
 `ingestExternalFile`  in C++ returns some error code, but the jni's `ingestExternalFile` returns `void` and is expected to throw an exception if there is any issue with the ingestion.

Reply:
Have added `deleteOldIngestion()` and `validateIngestion()` before and after `ingestExternalFile` that does check the number of files that got ingested.